web/src/beta/features/Navbar/index.tsx

```diff
@@ -51,9 +52,9 @@ const Navbar: React.FC<Props> = ({ sceneId, currentTab }) => {
 
 export default Navbar;
 
-const Wrapper = styled.div`
+const Wrapper = styled.div<{ theme: Theme }>`
```

Comment:
You can access theme without this.
`useTheme()` is only needed if you need to use theme properties inline.
You can do `${({ theme }) => theme.editorNavBar.bg}` inside any styled component

Reply:
Thank you, [fixed](https://github.com/reearth/reearth/pull/463/commits/aab265ece2dfbf5cb336bda4ce9994907a8a3660)!